Source/Bugsnag.h

```diff
@@ -217,12 +217,12 @@
 // =============================================================================
 
 /**
- * Replicates BugsnagConfiguration.context
- *
- * @param context A general summary of what was happening in the application
+ * Retrieves the context - a general summary of what was happening in the application
  */
 + (void)setContext:(NSString *_Nullable)context;
 
++ (NSString *_Nullable)getContext;
```

Comment:
I think this can be just `context`, no `get` required?

Reply:
Yep - I've amended this to be `context` instead.